models/core/core_models.yml

```
@@ -293,6 +293,8 @@ models:
         - encounters
       materialized: table
     columns:
+    - name: _dbt_source_relation
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)